src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -208,6 +208,10 @@ def __call__(self, poller):
         correlation_message = ''
         self.progress_controller.begin()
         correlation_id = None
+
+        base = azlogging.get_az_logger()
+        is_verbose = any(han.level == logs.INFO for han in base.handlers)
```

Comment:
'-v' is for the version if I'm not mistaken.  For '--verbose,' the levels for the handlers are determined when you configure the loggings.